test/fix-xml-string.test.js

```diff
@@ -0,0 +1,39 @@
+const assert = require('assert').strict;
+const fixXMLString = require('../lib/svg-sprite/fix-xml-string.js');
+
+describe('testing fixing svg string', () => {
+    it('should return valid svg file on svg with multiline attribute values', () => {
+        assert.equal(fixXMLString(`<svg viewBox="0 0 0 16
```

Comment:
Are you sure these are valid SVGs? SVGOMG doesn't parse it either, so SVGO will not parse it either.

Reply:
Yep, I checked it out in online XML and SVG validators

Reply:
An SVG file must have a `xmlns="http://www.w3.org/2000/svg"` namespace. See https://github.com/RazrFalcon/resvg/issues/192#issuecomment-569467534

If namespace is missing, it will report an error in resvg. I added a [test case in resvg-js](https://github.com/yisibl/resvg-js/pull/64).

Reply:
@yisibl hey! Thank!
This test is just for `fix-xml-string.js` which just fixes styles and incompletion in provided strings.

Reply:
OK